huaweicloud/services/apig/resource_huaweicloud_apig_api.go

```diff
@@ -863,29 +863,25 @@ func policyConditionSchemaResource() *schema.Resource {
 				Description: "The frontend authentication parameter name.",
 			},
 			"source": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  string(ConditionSourceParam),
-				ValidateFunc: validation.StringInSlice([]string{
-					string(ConditionSourceParam),
-					string(ConditionSourceSource),
-					string(ConditionSourceSystem),
-					string(ConditionSourceCookie),
-					string(ConditionSourceFrontendAuthorizer),
-				}, false),
+				Type:        schema.TypeString,
+				Optional:    true,
 				Description: "The type of the backend policy.",
 			},
 			"type": {
-				Type:     schema.TypeString,
-				Optional: true,
-				Default:  string(ConditionTypeEqual),
-				ValidateFunc: validation.StringInSlice([]string{
-					string(ConditionTypeEqual),
-					string(ConditionTypeEnumerated),
-					string(ConditionTypeMatching),
-				}, false),
+				Type:        schema.TypeString,
+				Optional:    true,
 				Description: "The condition type.",
 			},
+			"mapped_param_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The name of a parameter generated after orchestration.",
+			},
+			"mapped_param_location": {
```

Comment:
If there is no special reason, it is recommended to add "Computed = true" to the new fields "orchestrations", "mapped_param_name" and "mapped_param_location".

Reply:
there is not any value returned if the name and the location not set, and the computed behavior will occurs the update error because the old value carried.